i18n/fr/docusaurus-plugin-content-docs/version-23.04/alerts-notifications/resources-status.md

```diff
@@ -173,7 +173,7 @@ Cependant, vous pouvez faire une recherche beaucoup plus fine en utilisant le [C
 - **service_group** : rechercher les services rattachés à un groupe de services
 - **service_severity**: rechercher des services selon le nom de la criticité associée
 - **service_severity_level**: rechercher des services selon le niveau de la criticité associée
-- **s.description** : chercher uniquement sur la description du service (son nom dans la colonne **Resource**)
+- **s.description** : chercher uniquement sur le nom du service (indiqué dans la colonne **Resource**)
```

Comment:
While it is often displayed and refered as "name", it actually looks for the service description field in its configuration. (hence the name description)
I think we should keep "description" or maybe change to "champ description"